admin_manual/ai/app_assistant.rst

```diff
@@ -78,3 +78,83 @@ Context Chat
 In order to make use of our special Context Chat feature, offering in-context insights about your own data stored in Nextcloud, you will need the following apps:
 
 * :ref:`context_chat + context_chat_backend<ai-app-context_chat>` -  (Customer support available upon request)
+
+
+Configuration
+-------------
+
+The Assistant admin settings can be found under the "Artificial intelligence" section.
+You can disable the assistant top menu entry there. You can also disable the AI-related smart pickers.
+The commands to change the options are also listed in.
```

Comment:
the options are also listed in... ?

Reply:
don't remember what I was thinking when I wrote it but should mean the occ commands, I'll rephrase it.